lib/timely/rails/date_group.rb

```diff
@@ -2,12 +2,13 @@
 
 module Timely
   class DateGroup < ActiveRecord::Base
-    belongs_to :season, class_name: 'Timely::Season', optional: true
+    belongs_to :season, class_name: 'Timely::Season', optional: true, inverse_of: :date_groups
 
     weekdays_field :weekdays
 
     validates_presence_of :start_date, :end_date
     validate :validate_date_range!
+    validate :validate_weekdays_not_null
```

Comment:
Can we just use standard validation for this?
validates :weekdays, presence: true